js/src/lib/components/InferenceWidget/widgets/FillMaskWidget/FillMaskWidget.svelte

```diff
@@ -109,7 +113,7 @@
 			// entries = body -> text-classificartion
 			// entries = body[0] -> summarization
 			const entries = (
-				model.pipeline_tag === "text-classification" ? body[0] ?? [] : body
+				body?.[0] && Array.isArray(body[0]) ? body[0] : body
```

Comment:
Oh yeah, regarding this change: for some reason the data we get for "text-classification" is different between Endpoints and the Inference API. This change makes sure both response types are parsed correctly;